.circleci/config.yml

```diff
@@ -77,10 +35,84 @@ jobs:
           command: |
             cd drupal/web && php core/scripts/run-tests.sh --verbose --sqlite --module elasticsearch_helper
 
+jobs:
+  build:
+    docker:
+      - image: wunderio/circleci-codeception-base
+
+    steps:
+      - run:
+          name: Create Drupal
+          command: |
+            composer create-project drupal/recommended-project:8.8.6 drupal
+            cd drupal
+            COMPOSER_MEMORY_LIMIT=-1 composer require drupal/core-dev:^8
+            COMPOSER_MEMORY_LIMIT=-1 composer require drush/drush:9
+            COMPOSER_MEMORY_LIMIT=-1 composer require elasticsearch/elasticsearch:~6.0
+      - save_cache:
+          paths:
+            - ~/project/drupal
+          key: v2-composer-dependencies-{{ .Environment.CIRCLE_SHA1 }}
+
+  test-es-6:
+    docker:
+      - image: wunderio/circleci-codeception-base
+      - image: circleci/mariadb:10.2
+        environment:
+          MYSQL_DATABASE: drupal
+          MYSQL_ROOT_PASSWORD: dbpassword
+          MYSQL_ROOT_HOST: "%"
+      - image: docker.elastic.co/elasticsearch/elasticsearch:6.8.3
```

Comment:
Latest is 6.8.10 - should we always test against lates version or fexed version?